scripts/check_blocking_packages.py

```diff
@@ -0,0 +1,127 @@
+#!/usr/bin/env python
+
+import argparse
+
+import rosdistro
+import rosdistro.dependency_walker
+
+index = rosdistro.get_index(rosdistro.get_index_url())
+valid_distro_keys = index.distributions.keys()
+valid_distro_keys.sort()
+
+parser = argparse.ArgumentParser(description='')
+parser.add_argument('--distro', metavar='distribution', type=str,
```

Comment:
Please wrap after the open parenthesis to avoid the content dependent indentation of the following lines.